onnx/onnx.proto

```diff
@@ -671,6 +671,14 @@ message TypeProto {
     optional TypeProto value_type = 2;
   };
 
+  message SparseTensor {
+    // This field MUST NOT have the value of UNDEFINED
+    // This field MUST have a valid TensorProto.DataType value
+    // This field MUST be present for this version of the IR.
+    optional int32 elem_type = 1;
+    optional TensorShapeProto shape = 2;
```

Comment:
Can we also add the sparse tensor format to the type?

Reply:
The problem with the sparse tensor type format is that it would be encoded into the model. However, different backends may favor different ways to implement sparsity and, thus, may have different format sparsity. In fact, depending on the libraries used, different formats may be preferable on different devices.

Thus, it becomes a runtime property. The customer will still need to know what is the optimal format under the circumstances, however, hard-coding it into the model may not have the desired effect.